examples/example-oauth/src/main/java/io/grpc/examples/oauth/Constant.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.examples.oauth;
+
+import static io.grpc.Metadata.ASCII_STRING_MARSHALLER;
+
+import io.grpc.Context;
+import io.grpc.Metadata;
+
+/**
+ * Constants definition
+ */
+final class Constant {
+
+    static final String REFRESH_SUFFIX = "+1";
+    static final String ACCESS_TOKEN = "access-token";
+    static final Context.Key<String> CLIENT_ID_CONTEXT_KEY = Context.key("clientId");
+    static final Metadata.Key<String> AUTHORIZATION_METADATA_KEY = Metadata.Key.of("Authorization", ASCII_STRING_MARSHALLER);
+
+    private Constant() {
+        throw new AssertionError();
+    }
```

Comment:
nit: Don't really need this...

Reply:
Agreed. Copied from https://github.com/grpc/grpc-java/blob/master/examples/example-jwt-auth/src/main/java/io/grpc/examples/jwtauth/Constant.java